Task_objects/index.js

```diff
@@ -8,17 +8,35 @@ const cat = {
 };
```

Comment:
There is no declaration of the 'cat' variable before assigning a value to its 'name' property.